### DISCUSSION TITLE
Development

### DISCUSSION THREAD

**Discussion by jithendra945** [cuebook/CueObserve#69] [General]
![image](https://user-images.githubusercontent.com/83579160/127966263-2d4038b8-5517-4811-afda-f5edc7e533d0.png)
im getting this, when doing npm install.

any solutions ?

**Comment by sachinkbansal** [comment]
@vincue pls check

**Reply by jithendra945** [reply]
Please check the requirements, requirements like restframework, celerybeat are not there.

**Comment by vincue** [comment]
Hi @jithendra945, please ensure your node version is 12 or higher.

**Reply by jithendra945** [reply]
$ node --version
v16.4.1

**Reply by jithendra945** [reply]
by which version, it is working properly, so I can change to that

**Reply by vincue** [reply]
version 12

**Reply by jithendra945** [reply]
ok, i will try that.

**Reply by jithendra945** [reply]
any specific version
like 12.14.1 or something

**Reply by vincue** [reply]
yeah, that should work, please try on

**Reply by jithendra945** [reply]
yes, that works.

**Reply by jithendra945** [reply]
i trying to run this locally, anamoly detection task is not running.
i followed the https://cueobserve.cuebook.ai/development
